spectral_cube/io/casa_wcs.py

```diff
@@ -1,11 +1,57 @@
-import tempfile
 import numpy as np
 from astropy.wcs import WCS
+from astropy.io import fits
+from astropy.time import Time
 
 __all__ = ['wcs_casa2astropy']
 
+EQUATORIAL_SYSTEMS = ['B1950', 'B1950_VLA', 'J2000', 'ICRS']
 
-def wcs_casa2astropy(ndim, coordsys):
+SPECSYS = {}
+SPECSYS['BARY'] = 'BARYCENT'
+SPECSYS['TOPO'] = 'TOPOCENT'
+SPECSYS['LSRK'] = 'LSRK'
+SPECSYS['LSRD'] = 'LSRD'
+SPECSYS['GEO'] = 'GEOCENTR'
+SPECSYS['GALACTO'] = 'GALACTOC'
+SPECSYS['LGROUP'] = 'LOCALGRP'
+SPECSYS['CMB'] = 'CMBDIPOL'
+SPECSYS['REST'] = 'SOURCE'
+
+RADESYS = {}
+RADESYS['J2000'] = 'FK5'
+RADESYS['B1950'] = 'FK4'
+RADESYS['B1950_VLA'] = 'FK4'
+RADESYS['ICRS'] = 'ICRS'
+
+EQUINOX = {}
+EQUINOX['J2000'] = 2000.
+EQUINOX['B1950'] = 1950.
+EQUINOX['B1950_VLA'] = 1979.9
+
+AXES_TO_CTYPE = {
+    'Frequency': 'FREQ',
+    'GALACTIC': {'Longitude': 'GLON',
+                 'Latitude': 'GLAT'},
+    'SUPERGAL': {'Longitude': 'SLON',
+                 'Latitude': 'SLAT'},
+    'ECLIPTIC': {'Longitude': 'ELON',
+                 'Latitude': 'ELAT'}
+}
+
+for system in EQUATORIAL_SYSTEMS:
+    AXES_TO_CTYPE[system] = {'Right Ascension': 'RA--',
+                             'Declination': 'DEC-'}
+
+
+def sanitize_unit(unit):
+    if unit == "'":
+        return 'arcmin'
```

Comment:
do we need `"` for arcsec, or does that not appear?

Reply:
I haven't managed to find files that use this, but no harm in adding it, so I've done this.